astropy/coordinates/tests/test_solar_system.py

```diff
@@ -207,7 +208,8 @@ def setup(self):
     def test_erfa_planet(self, body, sep_tol, dist_tol):
         """Test predictions using erfa/plan94.
 
-        Accuracies are maximum deviations listed in erfa/plan94.c
+        Accuracies are maximum deviations listed in erfa/plan94.c, for Jupiter and
+        Mercury, and that quoted in Meeus "Astronomical Algorithms" (1998) for the Moon.
         """
         # Add uncertainty in position of Earth
         dist_tol = dist_tol + 1300 * u.km
```

Comment:
Is this needed for the Moon? I think I would just add it to `Mercury` in the list above as done for the other test (sorry, *I* should have done that!)